app/services/import/legacy/process-licence.service.js

```diff
@@ -6,7 +6,7 @@
  */
 
 const LicenceStructureValidator = require('../../../validators/import/licence-structure.validator.js')
-const PersistLicenceService = require('../persist-licence.service.js')
+const PersistLicenceService = require('../persist-import.service.js')
```

Comment:
If that is what we are naming the file we should name the value to match (and think this is the right way round!)

```suggestion
const PersistImportService = require('../persist-import.service.js')
```